VERSION

```diff
@@ -1 +1 @@
-1.5.0
+1.6.0
```

Comment:
Isn't it a bit early to release 1.6.0?

Reply:
The VERSION file in `master` branch contains the upcoming release version. CI uses it to publish pre-release versions (e.g. 1.6.0-adsf323) 

 We usually bump it as soon as the release branch is created but I forgot to do it after creating `release-1.5` branch.